include/term-slug.php

```diff
@@ -0,0 +1,186 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * @since 3.7
+ */
+class PLL_Term_Slug {
+
+	/**
+	 * @var PLL_Model
+	 */
+	private $model;
+
+	/**
+	 * @var PLL_Language
+	 */
+	private $lang;
+
+	/**
+	 * @var int
+	 */
+	private $parent;
+
+	/**
+	 * @var string
+	 */
+	private $taxonomy;
+
+	/**
+	 * @var string
+	 */
+	private $slug;
+
+	/**
+	 * @var string
+	 */
+	private $name;
+
+	/**
+	 * Constructor
+	 *
+	 * @since 3.7
+	 *
+	 * @param PLL_Model $model    Instance of the current PLL_Model.
+	 * @param string    $slug     The term slug.
+	 * @param string    $taxonomy The term taxonomy.
+	 * @param string    $name     The term name.*
+	 */
+	public function __construct( PLL_Model $model, $slug, $taxonomy, $name ) {
+		$this->model = $model;
+		$this->slug = $slug;
+		$this->taxonomy = $taxonomy;
+		$this->name = $name;
```

Comment:
Could you align the equal signs? My eyes are bleeding 😛

Reply:
Oops, yes I can 😁